src/data_retrieval/__main__.py

```diff
@@ -21,99 +22,192 @@
 parser_cds = subparsers.add_parser("cds", help="Copernicus CDS")
 parser_mars = subparsers.add_parser("mars", help="ECMWF MARS")
 
-parser_cds.add_argument("type", choices=["ecmwf", "era5"], help="Data types")
+parser_cds.add_argument(
+    "type", choices=["ecmwf", "era5"], help="Data types"
+)  # noqa: E501
```

Comment:
You do not need to add `# noqa: E501` after every single line. Please remove those that are not necessary